.pre-commit-config.yaml

```diff
@@ -13,12 +13,12 @@ repos:
       - id: check-added-large-files
   - repo: https://github.com/astral-sh/ruff-pre-commit
     # keep in sync with uv.lock
-    rev: 'v0.13.1'
+    rev: 'v0.13.3'
     hooks:
       - id: ruff
         args: [--fix, --exit-non-zero-on-fix]
-  - repo: https://github.com/psf/black
-    rev: 25.1.0
+  - repo: https://github.com/psf/black-pre-commit-mirror
```

Comment:
Apparently this is an official mirror that runs black faster, per https://github.com/psf/black/pull/3828/files